test/service/test_response.py

```diff
@@ -1442,7 +1442,8 @@ def test_ordering(self):
             hit_sort_values = {}
             for order, response in responses.items():
                 response.raise_for_status()
-                hit_sort_values[order] = [accessor(hit) for hit in response.json()['hits']]
+                hit_sort_values[order] = [accessor(hit) for hit in response.json()['hits'] if accessor(hit) is not None]
```

Comment:
line too long

Reply:
use filter()

Reply:
and map()

Reply:
Most importantly, why are we excluding None values from the asserted sort order? The sorting should be total and cover None values.

Reply:
Cant compare `Nonetype` and `int`,  might be better to replace `None` here with `float('-inf')` so we can keep ensure the sort comparisons.

Reply:
used none_safe_key